randomizer/randomize.py

```diff
@@ -103,7 +103,10 @@ def load_enemy_data(rom: Rom, items: Items) -> list:
     EnemyExtraData = namedtuple("EnemyExtraData",
                                 ["enemy_index", "name", "max_hp", "atk", "pdef", "mdef", "drop_chance", "drop_type",
                                  "drop_item"])
-    for item_data in load_tsv("data/EnemyData.tsv"):
+    file_name = "data/EnemyData.tsv"
+    if new_item:
+        fileName = "data/EnemyData_2.tsv"
```

Comment:
Can you change this to be the same as the way you did it in items.py?
```suggestion
    file_name = "data/ItemData_2.tsv" if new_item else "data/ItemData.tsv"
```

Reply:
oh yeah, that's an oversight